src/module.c

```diff
@@ -6066,14 +6074,12 @@ void RM_SetClientUser(RedisModuleClient *client, RedisModuleUser *user)
 /* Sets the persistent client object on the given context.  To unset the
  * client, call with the client seto NULL)
  */
-void RM_SetContextClient(RedisModuleCtx *ctx, RedisModuleClient *client)
-{
+void RM_SetContextClient(RedisModuleCtx *ctx, RedisModuleClient *client) {
     ctx->persistent_client = client;
 }
 
 /* Get the client flags that are exported to Redis Modules */
-uint64_t RM_GetClientFlags(RedisModuleClient *client)
-{
+uint64_t RM_GetClientFlags(RedisModuleClient *client) {
```

Comment:
the list of flags should be documented here.
i.e. we don't want to rely on the user looking in the header file for docs.
instead, these docs (comments above the APIs) are used to generate the reference doc.
https://redis.io/docs/reference/modules/modules-api-ref/

Reply:
added, copied GetContextFlags format